Lib/getpass.py

```diff
@@ -108,9 +126,15 @@ def win_getpass(prompt='Password: ', stream=None):
         if c == '\003':
             raise KeyboardInterrupt
         if c == '\b':
+            if echochar and pw:
+                msvcrt.putwch('\b')
+                msvcrt.putwch(' ')
+                msvcrt.putwch('\b')
```

Comment:
```suggestion
                msvcrt.putch('\b')
                msvcrt.putch(' ')
                msvcrt.putch('\b')
```

Those are not wide chars, so we don't need to use `putwch`.

Reply:
@picnixz I apologize again, as I was so eager to add this to 3.14, but I only have Windows now

the thing is that putch does not accept control characters, so you need to use putwch.
you can easily check this by simply trying to erase the character, and it will immediately give you an error:
```
msvcrt.putch('\b')
~~~~~~~~~~~~~^^^^^^
TypeError: putch() argument must be a byte string of length 1, not str
```

Reply:
Oh my bad!